content/rc/how-to/importing-data.md

```diff
@@ -5,61 +5,123 @@ weight: $weight
-import completes.
+{{% note %}}
+Expired keys are not imported.
+As a result, the number of keys in the source and destination databases can be different after the import is complete.
+{{% /note %}}
 
 ## Import a Dataset from a Redis Server
 
-To import a dataset from any publicly available Redis server, simply
-enter the following information in the Import Dataset dialog of your
-Redis Cloud instance and click the "Import" button when done:
+To import a dataset from any publicly available Redis server:
 
-1. Enter the hostname or the public IP address of the source Redis
-    server in the **Redis Hostname/IP Address** box.
-1. Enter the port of the source Redis server in the Port field, if it
-    is different from the default value of *6379*.
-1. If the source Redis server is set with a password, enter it in the
-    **Password** input control field.
+1. In the Redis Cloud management console, go to the database that you want to import into.
+1. Click **Import**.
+1. In the Source Type field, select **Redis**.
+1. In the **Redis Hostname/IP Address** field, enter the hostname or the public IP address of the source Redis server.
+1. In the **Port** field, enter the port of the source Redis server if it is not the default value of `6379`.
+1. If the source Redis server has a password, enter it in the **Password** field.
 
-## Import a Dataset from a RDB File
+## Import a Dataset from an RDB File
 
 You can import any standard RDB to your Redis Cloud instance.
-Both uncompressed and compressed (with a .gz suffix) RDB files are
-acceptable.
+Both uncompressed and compressed (with a .gz suffix) RDB files are valid.
 
 ### RDB File from an FTP or HTTP Server
 
-To import a dataset from an RDB file that is accessible via a public FTP
-or HTTP server, enter the file's full URL in the **Import Dataset**
-dialog's **RDB File Path** field. Use the following standard pattern to
-provide the file's URL:
+To import an RDB file that is stored in an FTP or HTTP server:
+
+1. In the Redis Cloud management console, select the database that you want to import into.
+1. Click **Import**.
+1. Enter the details for the RDB file:
+    - Source Type - Select **FTP** or **HTTP**.
+    - RDB file path - Enter the URL for RDB file: `<protocol>://[username][:password]@[:port]/[path/]filename.rdb[.gz]`
+
+    Where:
+
+    - `protocol` - Server protocol: ftp, ftps, http, https
+    - `username` - Your username, if necessary
+    - `password` - Your password, if necessary
+    - `hostname` - Hostname or IP address of the server
+    - `port` - Port number of the server, if not `6379`
+    - `path` - Path to the file, if necessary
+    - `filename` - Filename of the RDB file, including the .gz suffix if the file is compressed
+
+### RDB File From an Amazon Simple Storage Service (AWS S3) Bucket
+
+Before you import the RDB file through the Redis Cloud Essentials management console, you must share the file from the AWS management console.
```

Comment:
If this is a prerequisite for the previous procedure, should we call move it to earlier on the page? Also, call it out as a prerequisite in the preceding procedure.